README.md

```diff
@@ -148,7 +148,7 @@ some combinations from making sense: for example, searching for a Python `class`
 however:
 
 ```console
-$ cat birds.py | srgn --py 'class' --py 'doc-strings' # From earlier example; note `--py` is a shorthand for `--python`
```

Comment:
This stopped working due to 

https://github.com/alexpovel/srgn/pull/168/files#diff-c4682009f01c1c630ec1481e9dae8b61781728c8aa12f1f9000abea84f1cf810R522-R524

?

Can you remove unused new entries such as `tag("rust-query-file"),`, then comment in `tag("py")` etc. again? That way we get both worlds working (not touching existing docs), at the cost of not having 1:1 compatibility between `readme.rs` and the actual CLI. Doesn't seem like a high price to pay (implementers of new README examples might run into bizarre errors... I'll document the "here be dragons" part of this code base better)

Reply:
I reverted the doc changes in cdc04bc702931e1dfbf8dd1f16756433ecdbcc89 and added all `--[lang-query-file]` args to the `readme.rs` parser in an `alt` combinator like the rest of the args in a77b149274f70f288f21f490464478f751e6af5b.